controllers/front/ProductController.php

```diff
@@ -710,7 +711,7 @@ protected function formTargetFormat()
         $this->context->smarty->assign('customizationFormTarget', $customization_form_target);
     }
 
-    protected function formatQuantityDiscounts($specific_prices, $price, $tax_rate, $ecotax_amount)
```

Comment:
Indeed, I agree with @aleeks, we can't change the function signature. Could you amend your commit to keep the parameter, please? Even if we won't use it anymore.

Regards